SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -78,6 +78,7 @@ extension SKPaymentTransactionState: CustomDebugStringConvertible {
         case .failed: return "failed"
         case .restored: return "restored"
         case .deferred: return "deferred"
+        @unknown default: return "default"
```

Comment:
Personally I think this would be better as `unknown`?

```suggestion
        @unknown default: return "unknown"
```